pkg/micropython/include/micropython.h

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (C) 2019 Kaspar Schleiser <kaspar@schleiser.de>
+ *               2019 Inria
+ *               2019 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     pkg_micropython
+ *
+ * @{
+ *
+ * @file
+ * @brief       MicroPython RIOT specific API
+ *
+ * @author      Kaspar Schleiser <kaspar@schleiser.de>
+ */
+
+#ifndef MICROPYTHON_H
+#define MICROPYTHON_H
+
+#include <stdint.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#ifndef MP_RIOT_HEAPSIZE
+#define MP_RIOT_HEAPSIZE (16U*1024)
```

Comment:
Could you add some documentation on this heap size? What value makes sense for an application? Maybe a pointer to micropython docs. Could be in the package doc.txt